<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 70, 85);</string>
   </property>
   <widget class="QFrame" name="frm_header">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>263</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>-8</y>
       <width>260</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../assets/images/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frm_body">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>83</y>
      <width>301</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="Pages">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(35, 54, 66);</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_home">
        <widget class="QLabel" name="lbl_greeting">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>18</y>
           <width>241</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Olá, Tacyana</string>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QFrame" name="frame">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>242</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 233, 138, 0.6);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>15</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>Conta Corrente</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_acount_balance">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>160</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>R$ 1.350,00</string>
          </property>
         </widget>
        </widget>
        <widget class="QFrame" name="frame_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>242</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 233, 138, 0.6);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>15</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>Cartão de crédito</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_credit_balance">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>160</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>R$ 0,00</string>
          </property>
         </widget>
        </widget>
        <widget class="QFrame" name="frame_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>242</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 233, 138, 0.6);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>15</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>Investimentos</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_investment_balance">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>160</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>R$ 0,00</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_deposit">
        <widget class="QFrame" name="frame_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>242</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 233, 138, 0.6);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>15</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>Saldo disponível</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_acc_balance_deposit">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>160</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>R$ 1.350,00</string>
          </property>
         </widget>
        </widget>
        <widget class="QLineEdit" name="txt_deposit_amount">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
padding-left: 5 px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>184</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Valor a depositar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_deposit_amount">
         <property name="geometry">
          <rect>
           <x>156</x>
           <y>270</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(63, 97, 118);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(255, 233, 138);
	
	color: rgb(45, 70, 85);
}</string>
         </property>
         <property name="text">
          <string>Depositar</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Depósito</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_withdraw">
        <widget class="QFrame" name="frame_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>242</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 233, 138, 0.6);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>15</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>Saldo disponível</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_acc_balance_withdraw">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>160</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string>R$ 1.350,00</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>184</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Valor a sacar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_withdraw_amount">
         <property name="geometry">
          <rect>
           <x>156</x>
           <y>270</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(63, 97, 118);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(255, 233, 138);
	
	color: rgb(45, 70, 85);
}</string>
         </property>
         <property name="text">
          <string>Sacar</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_withdraw_amount">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
padding-left: 5 px;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Saque</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frm_nav">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>435</y>
      <width>280</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>17</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_home">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-radius: 10px;
}

QPushButton:hover {
	
	background-color: rgb(63, 97, 118);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../assets/images/home.png</normaloff>../assets/images/home.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_deposit">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-radius: 10px;
}

QPushButton:hover {
	
	background-color: rgb(63, 97, 118);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../assets/images/deposit.png</normaloff>../assets/images/deposit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_withdraw">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-radius: 10px;
}

QPushButton:hover {
	
	background-color: rgb(63, 97, 118);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../assets/images/withdraw.png</normaloff>../assets/images/withdraw.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
